x-pack/plugins/infra/public/components/logging/log_text_stream/loading_item_view.tsx

```diff
@@ -39,13 +40,26 @@ export class LogTextStreamLoadingItemView extends React.PureComponent<
       return (
         <ProgressEntry alignment={alignment} className={className} color="primary" isLoading={true}>
           <ProgressMessage>
-            <EuiText color="subdued">Streaming new entries</EuiText>
+            <EuiText color="subdued">
+              <FormattedMessage
+                id="xpack.infra.logs.streamingNewEntriesText"
+                defaultMessage="Streaming new entries"
+              />
+            </EuiText>
           </ProgressMessage>
           {lastStreamingUpdate ? (
             <ProgressMessage>
               <EuiText color="subdued">
-                <EuiIcon type="clock" /> last updated{' '}
-                <RelativeTime time={lastStreamingUpdate} refreshInterval={1000} /> ago
+                <FormattedMessage
+                  id="xpack.infra.logs.lastStreamingUpdateText"
+                  defaultMessage="{clockIcon} last updated {lastUpdateTime} ago"
+                  values={{
+                    clockIcon: <EuiIcon type="clock" />,
+                    lastUpdateTime: (
+                      <RelativeTime time={lastStreamingUpdate} refreshInterval={1000} />
```

Comment:
I think `now` is fine too :+1:

Reply:
@weltenwort Sorry, but we cannot use `FormattedRelative` for now, because the locale data for relative time is currently loaded just for `en` locale. We need to figure out when and where to load another locale data for relative time. As soon as I will fix it I'll get back to you. Can I leave this translation as it is for now?

Reply:
Sure, but it seems the interpolation of the nested React components doesn't quite work:

![image](https://user-images.githubusercontent.com/973741/48836906-468e5880-ed84-11e8-9392-d9a2f98410a4.png)

Reply:
@weltenwort Yes, this is because of pseudo locale. Pseudo locale  works in the way it translates values as it is. But if you use custom local it will not translate values so all will be OK. I prepered an example with fr locale:
![image](https://user-images.githubusercontent.com/23155410/48906386-58035d80-ee75-11e8-8390-9f6bc19fcabf.png)